ios/RNSScreenStack.mm

```diff
@@ -71,14 +99,38 @@ @implementation RNSPanGestureRecognizer
 @implementation RNSScreenStackView {
   UINavigationController *_controller;
   NSMutableArray<RNSScreenView *> *_reactSubviews;
-  __weak RNSScreenStackManager *_manager;
-  BOOL _hasLayout;
   BOOL _invalidated;
+  BOOL _isFullWidthSwiping;
   UIPercentDrivenInteractiveTransition *_interactionController;
+  BOOL _hasLayout;
+  __weak RNSScreenStackManager *_manager;
+#ifdef RN_FABRIC_ENABLED
+  UIView *_snapshot;
+#else
   BOOL _updateScheduled;
-  BOOL _isFullWidthSwiping;
+#endif
 }
 
+#ifdef RN_FABRIC_ENABLED
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if (self = [super initWithFrame:frame]) {
+    static const auto defaultProps = std::make_shared<const facebook::react::RNSScreenStackProps>();
+    _props = defaultProps;
+    _reactSubviews = [NSMutableArray new];
```

Comment:
Could you move those common initializations to a method similar to the one in `RNSScreen`?